debian/changelog

```diff
@@ -1,5 +1,9 @@
 sonic (1.0.0) stable; urgency=medium
 
+  * next-hop key add ifname
```

Comment:
please keep the changelog as it is since right now we're not updating this file routinely.

Reply:
New version dpkg-deb use the newest time in chaneglog to set deb files timestamp. There is a problem that new files with same size and timestamp cannot be saved to image in process of PR build, so in this PR fpmsynd cannot be updated and many test cases failed. I add changelog just for building swss.deb with newer timestamp and pass tests.
THE PROBLEM MUST BE RESOLVED, or else the VS test become meaningless, for the running version may not be the codes compiled.

Reply:
are you referring to the VS test running under this pull request? and the fpmsyncd is not replaced with the latest binary? if that is the case as you described, I'll check.

Reply:
Yes，I have forward the email which describes this issue detailly to you. It is common issue, not  just on this PR.

Reply:
could you check right now if the binaries could overwrite the old ones?
I have added https://github.com/Azure/sonic-build-tools/commit/e9e39576e2f964538cb6648cf3127017b6745f2a to fix this issue.